clang/docs/ReleaseNotes.rst

```diff
@@ -151,6 +151,8 @@ C++ Language Changes
 - The builtin type alias ``__builtin_common_type`` has been added to improve the
   performance of ``std::common_type``.
 
+- During the lookup for a base class name, non-type names are ignored.
+
```

Comment:
This shouldn’t be here since this is a bug fix. There’s a ‘bug fixes to c++ support’ section or whatever it was called below where this should go.

Reply:
Sorry I did not found this originally. I just saw the first C++ section that has matched the topic and put the note there. Moved it to the suggested place.